.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "Ubuntu",
+    "image": "mcr.microsoft.com/devcontainers/base:jammy",
+
+    // Features to add to the dev container. More info: https://containers.dev/implementors/features.
+    "features": {
+        "ghcr.io/devcontainers/features/github-cli:1": {},
+        "ghcr.io/devcontainers-contrib/features/pre-commit:1": {},
+        "./serenity": {
+            "llvm_version": 15,
+            "enable_ladybird": true,
+            "enable_serenity": true
+        },
+        "ghcr.io/devcontainers/features/desktop-lite": {
+            "password": "vscode",
+            "webPort": "6080",
+            "vncPort": "5901"
+        }
+    },
+
+    // Use 'forwardPorts' to make a list of ports inside the container available locally.
+    "forwardPorts": [6080, 5901],
+    "portsAttributes": {
+        "5901": {
+            "label": "VNC"
+        },
+        "6080": {
+            "label": "WEB VNC"
```

Comment:
Does this need to be all caps or is `Web VNC` ok?

Reply:
Nop no requirement, just copy pasta from the action I referenced for how to get vnc into the devcontainer working.